hack/kind-install-for-capd.sh

```diff
@@ -26,19 +26,11 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
-NAME="capi-test"
-
-while [[ $# -gt 0 ]]; do
-  key="$1"
-  case $key in
-    -n|--name)
-      NAME="$2"
-      shift
-      shift
-      ;;
-  esac
-done
-
+if [[ -z "$CAPI_KIND_CLUSTER_NAME" ]]; then
```

Comment:
I know there's barely code left but we can simplify this further to:
```bash
NAME=${CAPI_KIND_CLUSTER_NAME:-"capi-test"}
```

Reply:
done